app/utils/Logger/index.js

```diff
@@ -1,5 +1,5 @@
 const pino = require('pino')
-const env = require('./env')
+const env = require('../../env')
```

Comment:
not sure I agree with this move

Reply:
folder structure helps to unit test and wrap module/util specific functionality, also it's easier to test